publish-docs.js

```diff
@@ -161,13 +151,6 @@ function build() {
         };
         replace.sync(options4);
 
-        const options5 = {
-            files: ['src/index.mdx'],
-            from: /<version\\>/g,
-            to: `${version}`,
-        };
-        replace.sync(options5);
```

Comment:
Needed to update the version here:
![image](https://user-images.githubusercontent.com/154725/94915600-742ea780-04ad-11eb-9cf8-0fe39ee046b3.png)

(But had to be changed from `index.mdx` to `index.md`. Will push fix soon!)

Reply:
Also, while we're at it, the info shown there (under *Home/Lime Elements*) I would quite like to see on the landing page instead.

Reply:
Yeah, great input!
I think I just used the readme in the beginning when there was no support for separate md-files, but you should definitely be able to add something else as the landing page